readme.txt

```diff
@@ -22,6 +22,11 @@ Here is how to get started:
 
 Then open [a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/) with the suggested changes.
 
+== How to use ==
+Install and activate the plugin.
```

Comment:
Should we also suggest how to find the plugin in the WordPress dashboard? Something like `Search for "Two Factor" under "Plugins" → "Add New" in your WordPress dashboard to install the plugin.`?

Secondly, it looks like an ordered list. Could we style it like that and add an empty line after the heading to match the rest of the sections.

```
== How to Use ==

- Search for "Two Factor" under "Plugins" → "Add New" in your WordPress dashboard to install the plugin.
- Go to "Users" → "Your Profile" and choose the authentication method best to you.
- That's all.
```

Reply:
Well, this could be an option, however, I never got these kind of instructions. It's listed on the official wordpress.org website so people directly assume they can install it from Plugins > Add new. Or am i wrong?

Reply:
Honestly, I'm not sure. First time users might not know this so adding it for extra clarity might help some. Others can just skip it.

There is also the ["Installation" section](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#installation) for `readme.txt` -- maybe put it all there?

I guess the most important part is the `"Users" → "Your Profile"` instruction.

Reply:
Well, both could work together?
Exactly, it took me about 10min to figure that one out.